src/devtools/client/inspector/markup/actions/markup.ts

```diff
@@ -50,12 +56,26 @@ export function reset(): ResetAction {
 /**
  * Clears the tree and adds the new root node.
  */
-export function newRoot(rootNodeFront: NodeFront): UIThunkAction {
+export function newRoot(): UIThunkAction {
   return async ({ dispatch }) => {
+    const pause = ThreadFront.currentPause;
```

Comment:
forgot if we do this now, but passing threadFront through as an option would be nice


```js
return async ({ dispatch, threadFront }) => {
}

```

Reply:
No, this currently isn't added to the thunk args.